src/firebase/firebase_auth.cr

```diff
@@ -108,6 +140,35 @@ module Google
       query perform(query_request(expression, **opts))
     end
 
+    # API details: https://cloud.google.com/identity-platform/docs/reference/rest/v1/projects.accounts/update
+    def update_request(local_id : String, display_name : String? = nil, email : String? = nil, password : String? = nil, disable : Bool? = nil, **opts)
+      opts = opts.merge({
+        localId: local_id,
+      })
+      opts = opts.merge({displayName: display_name}) if display_name
+      opts = opts.merge({email: email}) if email
+      opts = opts.merge({password: password}) if password
+      opts = opts.merge({disableUser: disable}) if disable
```

Comment:
```suggestion
      opts = opts.merge({disableUser: disable_user}) unless disable_user.nil?
```

Passing `false` wouldn't work before.
I also changed it to `disabled_user` to be closer to the API description.